src/Microsoft.OData.Edm/Parameterized.Microsoft.OData.Edm.cs

```diff
@@ -18,2154 +18,2506 @@ internal static class Strings {
         /// <summary>
         /// A string like "Unexpected primitive type kind."
         /// </summary>
-        internal static string EdmPrimitive_UnexpectedKind {
-            get {
+        internal static string EdmPrimitive_UnexpectedKind
+        {
+            get
+            {
                 return Microsoft.OData.Edm.EdmRes.GetString(Microsoft.OData.Edm.EdmRes.EdmPrimitive_UnexpectedKind);
             }
         }
 
         /// <summary>
         /// A string like "Unexpected path type kind."
         /// </summary>
-        internal static string EdmPath_UnexpectedKind {
-            get {
+        internal static string EdmPath_UnexpectedKind
+        {
+            get
+            {
                 return Microsoft.OData.Edm.EdmRes.GetString(Microsoft.OData.Edm.EdmRes.EdmPath_UnexpectedKind);
             }
         }
 
         /// <summary>
         /// A string like "Annotation of type '{0}' cannot be interpreted as '{1}'."
         /// </summary>
-        internal static string Annotations_TypeMismatch(object p0, object p1) {
+        internal static string Annotations_TypeMismatch(object p0, object p1)
+        {
             return Microsoft.OData.Edm.EdmRes.GetString(Microsoft.OData.Edm.EdmRes.Annotations_TypeMismatch, p0, p1);
         }
 
         /// <summary>
         /// A string like "The annotation must have non-null target."
         /// </summary>
-        internal static string Constructable_VocabularyAnnotationMustHaveTarget {
-            get {
+        internal static string Constructable_VocabularyAnnotationMustHaveTarget
+        {
+            get
+            {
                 return Microsoft.OData.Edm.EdmRes.GetString(Microsoft.OData.Edm.EdmRes.Constructable_VocabularyAnnotationMustHaveTarget);
             }
         }
 
         /// <summary>
-        /// A string like "An entity type or a collection of an entity type or a complex type is expected."
+        /// A string like "An entity type or a collection of an entity type is expected."

```

Comment:
Why did this string change?

Reply:
This file is auto generated. I simply executed the tt script. I don't know what else to tell you. :)